oca_dependencies.txt

```diff
@@ -1,2 +1,3 @@
 account-financial-reporting
 partner-contact
+edi
```

Comment:
`base_ubl` depends in turn on `account_tax_unece` which is in `community_data_files`. Should that also be included here?

Reply:
Travis and Runbot are green, so it seems not needed. Otherwise we would see an error on both while installing the module.